build: release: v1.16.0-rc2 (#8884)
filecoin-project/lotus: CHANGELOG.md

# Lotus changelog

# 1.16.0-rc2 / 2022-06-22

> This is the second release of the upcoming MANDATORY release of Lotus v1.16.0 that supports Filecoin network v16, codenamed the Skyr upgrade.

Reviewer: ```suggestion
This is the second release candidate for the upcoming MANDATORY release of Lotus v1.16.0 that supports Filecoin network v16, codenamed the Skyr upgrade.
```

Filecoin nv16 upgrade will introduce non-programmable FVM(FVM m1) and switch the network from using go spec-actor to rust built-in actor. Full changelog will be published upon final release.

## Mainnet nv16 Skyr Upgrade

This release candidate sets mainnet upgrade at epoch 1044660, 2022-06-16T17:30:00Z. The bundle that the network will be using is v8.0.0, manifest CID will be included upon final release.

> Your lotus node will switch from legacy VM to FVM atomically upon the upgrade. Easily enable envvar `LOTUS_USE_FVM_TO_SYNC_MAINNET_V15` to sync calibration using FVM to sync network v15.

Reviewer: ```suggestion
Your lotus node will switch from the Legacy VM (that depended on go-based specs-actors) to FVM (that uses Rust-based builtin-actors) atomically upon the upgrade. 
```

Reviewer: I don't think there's any real use case for running v15 through FVM any more.

## Calibration-net Upgrade

This release candidate sets the caibration-net upgrade at epoch 1044660, 2022-06-16T17:30:00Z. The bundle that the network should be using is v8.0.0-rc.1(located at `build/actors/v8.tar.zst` ) upon migration, manifest CID `bafy2bzacedrdn6z3z7xz7lx4wll3tlgktirhllzqxb766dxpaqp3ukxsjfsba`.

## Between rc2 & rc1

> - We fixed a execution trace bug in ffi for implicit messages, and the execution trace should be more alike legacy VM ones now!

Reviewer: ```suggestion
- We fixed a execution trace bug in ffi for implicit messages, and the execution trace should be more like legacy VM ones now!
```

> - Your market node will now perform a light migration for migrate non-utf deal labels incorporation with FIP-0027.

Reviewer: ```suggestion
- Your market node will now perform a light migration to migrate non-utf deal labels, as needed by FIP-0027.
```
